client/app/reducers.js

```diff
@@ -4,10 +4,12 @@ import { FAQ_LOADED } from './actions';
 // Retrieve FAQ List from server
 const currentQuestions = (state = [], action) => {
 	switch (action.type) {
-		case FAQ_LOADED:
+		case FAQ_LOADED: {
```

Comment:
We do this because of lexical scoping reasons. Variables declared that don't use a curly bracket in a switch statement may leak. https://eslint.org/docs/rules/no-case-declarations